org-formation/organization-tasks.yaml

```diff
@@ -1,13 +1,34 @@
+# TIP: keep your tasks organized!
+# group your tasks by feature and assign each feature a folder.
+# use this file to include the various features and manage dependencies between features using DependsOn.
+# use a numbering scheme to prefix the feature folders. this way you can control the order in which the folders are listed.
+# e.g:
+#   /000-organization-build/{organization-tasks.yml & templates}
+#   /100-private-types/{organization-tasks.yml & templates}
+#   /200-account-defaults/{organization-tasks.yml & templates}
+#   /210-budget-alarms/{organization-tasks.yml & templates}
+#   /300-iam/{organization-tasks.yml & templates}
+#   /400-sso/{organization-tasks.yml & templates}
+#   /500-cloudtrail/{organization-tasks.yml & templates}
+#   /600-config/{organization-tasks.yml & templates}
+#   /610-config-custom-rules/{organization-tasks.yml & templates}
+#   /650-config-conformance-packs/{organization-tasks.yml & templates}
+#   /660-config-security-hub/{organization-tasks.yml & templates}
+#   /###-etc/
+#   /organization-tasks.yml <-- this file
+#   /organization-parameters.yml <-- contains global parameter values that can be included
+#   /organization.yml <-- contains your organization model
+
 OrganizationUpdate:
   Type: update-organization
   Template: ./organization.yaml
 
-CommunityIamPasswordPolicyRP:
-  Type: register-type
-  SchemaHandlerPackage: s3://community-resource-provider-catalog/community-iam-passwordpolicy-0.2.0.zip
-  ResourceType: 'Community::IAM::PasswordPolicy'
-  MaxConcurrentTasks: 10
-  OrganizationBinding:
-    IncludeMasterAccount: true
-    Account: '*'
-    Region: us-east-1
+Baseline:
+  Type: include
+  Path: ./000-baseline/organization-tasks.yml
```

Comment:
I'm not sure how this helps with my goal of setting consistent password policy across all my accounts.  I think what I want is to execute Baseline on all my account while only executing OrganizationUpdate  on my management account?   However since this file couples OrganizationUpdate and Baseline so I can't selectively execute them on separate accounts?